staging/src/k8s.io/apimachinery/pkg/api/apitesting/roundtrip/fuzz_norace.go

```diff
@@ -0,0 +1,26 @@
+//go:build !race
+// +build !race
+
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package roundtrip
+
+import (
+	flag "github.com/spf13/pflag"
+)
+
```

Comment:
Tiny nit / suggestion: maybe it makes more sense to only put the default value in this file (a const?) and leave the rest of the flag definition common?